conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/AbstractFeignJaxRsClientBuilder.java

```diff
@@ -97,6 +98,7 @@ public final AbstractFeignJaxRsClientBuilder hostEventsSink(HostEventsSink newHo
                                                 cborObjectMapper,
                                                 new JacksonEncoder(objectMapper)))))
                 .decoder(createDecoder(objectMapper, cborObjectMapper))
+                .errorDecoder(QosErrorDecoder.INSTANCE)
```

Comment:
I'd kinda prefer to pass in `new ErrorDecoder.Default()` to somewhat match the delegation chain above - that way it's clear we're just adding one tiny bit of behaviour, not overriding all of it.